protocols/peer/contracts/Peer.sol

```diff
@@ -33,6 +33,9 @@ contract Peer is IPeer, Ownable {
   // Swap contract to be used to settle trades
   ISwap public swapContract;
 
+  // The address holding tokens that will be trading through this peer
+  address private tradeWalletAddress;
```

Comment:
make this _tradeWallet as it's a private variable and keep it consistent with its getter function